docs/prerequisites/gro-api-corporate-proxy.rst

```diff
@@ -0,0 +1,77 @@
+Using Gro API behind a Corporate Proxy
+#######################################
+
+The Gro API uses the requests module to communicate with the Gro Servers. If you are behind a corporate proxy, you may see the error message below:
+ 
+.. code-block:: python
+
+	requests.exceptions.ProxyError: HTTPSConnectionPool … (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 407 Proxy Authentication Required')))
+
+Set an Environment Variable
+===========================
+
+Note: The variables in the examples below should be replaced with the actual values for your environment.
+
+Windows
+-------
+
+To change environment variables on Windows:
+#. In the Start menu, search for “env”.
+#. Select “Edit Environment Variables for your account”
+#. Select “Environment Variables…”
+#. Press “New…”
+#. Add two variables :code:`http_proxy` and :code:`https_proxy` both with the same value depending on the type of proxy:
+	#. Unauthenticated Proxy:
+		#. http_proxy="http://your-proxy-domain:<port>"

```

Comment:
For each of these, let's use `:code:` wrapping, since otherwise the quotes are going to mess people up. Quotes rendered in the browser don't copy/paste into text editors well.

Reply:
done, I think